spec/models/collection_spec.rb

```diff
@@ -4,96 +4,104 @@
 
 RSpec.describe Collection do
   let(:user) { create(:user) }
-  let(:collection) { described_class.create(title: collection_title_fixture, user:) }
+  let(:collection) { create(:collection, title: collection_title_fixture, user:) }
 
   before do
     allow(Notifier).to receive(:publish)
   end
 
-  describe 'Add participants to a collection' do
```

Comment:
I judged this an unnecessary `describe` block. Just makes the spec output noisier.